jetty-start/src/main/java/org/eclipse/jetty/start/StartArgs.java

```diff
@@ -1505,6 +1505,13 @@ public void setProperty(String key, String value, String source)
             {
                 JavaVersion ver = JavaVersion.parse(value);
                 properties.setProperty("java.version.platform", Integer.toString(ver.getPlatform()), source);
+
+                // features built into java.
+                // In Jetty 10+ these will always be true, but still need to stick around for users that
+                // want to move between Jetty 9.4.x and 10.0.x+
+                properties.setProperty("runtime.feature.alpn", Boolean.toString(isMethodAvailable(javax.net.ssl.SSLParameters.class, "getApplicationProtocols", null)), source);
+                properties.setProperty("runtime.feature.jpms", Boolean.toString(isClassAvailable("java.lang.ModuleLayer")), source);
```

Comment:
`*.feature.jpms` is not used, so I would remove it -- no need to leave in the code unnecessary stuff.